edgelet/edgelet-http/src/error.rs

```diff
@@ -148,6 +165,22 @@ impl From<Context<ErrorKind>> for Error {
     }
 }
 
+impl From<NativeTlsError> for Error {
+    fn from(error: NativeTlsError) -> Self {
+        Error {
+            inner: error.context(ErrorKind::NativeTls),
+        }
+    }
+}
+
+impl From<ErrorStack> for Error {
+    fn from(error: ErrorStack) -> Self {
+        Error {
+            inner: error.context(ErrorKind::OpenSSLPkcs12),
+        }
+    }
+}
```

Comment:
I don't think we are doing these `From` impls any more. @arsing to confirm

Reply:
Yes.

Reply:
Fixed. Thanks for your comment.